scripts/start_update.sh

```diff
@@ -102,6 +104,28 @@ extract_update_rootfs()
     return 0
```

Comment:
Shouldn't we unmount ${UPDATE_MOUNT} here? As we no longer need it, and if it's still mounted when we are going to resize we might run into issues. Not sure if the resize script can unmount it from the chroot if it is mounted outside the chroot...